src/main/java/com/entropyteam/entropay/employees/controllers/ContractController.java

```diff
@@ -14,10 +13,20 @@
 @CrossOrigin
 @RequestMapping(value = "/contracts", produces = MediaType.APPLICATION_JSON_VALUE)
 public class ContractController extends BaseController<ContractDto, UUID> {
-
+    private final ContractService contractService;
     @Autowired
     public ContractController(ContractService contractService) {
         super(contractService);
+        this.contractService = contractService;
+    }
+
+    @PostMapping("/v2")
+    public ResponseEntity<ContractDto> create(@RequestBody ContractDto contract) {
+        return ResponseEntity.ok(contractService.create(contract));
     }
 
+    @PatchMapping("/{id}")
+    public ResponseEntity<ContractDto> modifyStatus(@PathVariable UUID id, @RequestParam boolean active) {
+        return ResponseEntity.ok(contractService.modifyStatus(id, active));
+    }
```

Comment:
I would change to @PutMapping("/{id}/activate"), so that, the endpoint will activate the requested contract and deactivate the rest of them.

Reply:
I did it this way because of this use case written in the ticket:
_Inactive a contract. So user does not have any active contract_

Reply:
Good catch Juan, so i would change to: @PutMapping("/{id}/deactivate"), what do you think?

Reply:
We need to have to ability to activate and deactivate any contract, that's why were are passing the boolean here, not to create two different endpoints. What do you prefer we do here?